js/components/fetch-token.mjs

```diff
@@ -0,0 +1,144 @@
+/** 
+ * Get accessToken from local storage
+ * Create functions to API with header included token
+ */
+const givenToken = localStorage.getItem("accessToken");
+
+// GET header
+const getData = {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${givenToken}`,
+    },
+  };
+
+
+
+/**
+ * Get all Feed posts
+ * @date 2023-10-12
+ * @param {string} url This is an API link to get posts
+ * @returns {Array} This is an array of post objects. Later using these as param in showPosts function
+ */
+async function getPosts(url) {
+try {
+    getData;
+    const response = await fetch(url, getData);
+    const posts = await response.json();
+    return posts;
```

Comment:
```suggestion
    if (response.ok) {
      return posts;
    } else {
      throw new Error("Could not get posts!")
    }
```

Combined with my other comment, this will help to improve your error handling.

Reply:
I fix these !